MAINTENANCE.md

```diff
@@ -0,0 +1,89 @@
+# Roles, People, and Processes for Maintaining the Tagging Schema
+
+This document outlines how this project is maintained.
+
+## Roles & People
+
+### Maintainer Role
+
+[Martin](https://github.com/tyrasd) maintains this project as part of his work on the iD Editor project.
+
+The maintainer role includes:
+- Having the final say in decisions for the tagging schema.
+- Creating releases.
+- Updating dependencies.
+- Assign roles.
+
+and all the following roles.
+
+Github shows a `(Member)` label next to users with full access to this repository and the organisation.
+
+### Contributor Role
+
+- [Kyle](https://github.com/k-yle) contributes to this project as a volunteer.
+- [Tobias](https://github.com/tordans) contributes to this project as a volunteer.
+
+The contributor role includes:
+- Reviewing PRs.
+- Merging "clear-cut" PRs.
```

Comment:
Is it OK for them to merge own PRs if they seem clear cut for them?

Reply:
Technically they cannot due to branch protection rules. But lets make it explicit. How about…

```suggestion
- Merging "clear-cut" PRs by others.
```